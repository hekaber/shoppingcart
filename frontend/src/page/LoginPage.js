import React from 'react';
import {Input} from 'reactstrap';

export default class LoginPage extends React.Component {

    render() {
        return (
            <div className="login-container">
                <Input type="username"
                       name="username"
                       placeholder="example@example.com hello    s"/>
            </div>
        )
    }
}
