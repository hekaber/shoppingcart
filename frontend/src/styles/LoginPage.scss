.login-form{

  width: 40%;
  margin: 10% auto;

  ::placeholder{
    color: lightgrey;
  }

  .btn .btn-primary{
    width: 40%;
  }

  .btn-secondary{
    background-color: #007bff;
    border-color: #007bff;
  }
}
