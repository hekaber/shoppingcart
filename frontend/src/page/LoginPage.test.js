import React from 'react';
import { shallow } from "enzyme";
import LoginPage from "./LoginPage"

describe('Login Page', function () {

    const defaultProps = {
        baseUrl: "/"
    };

    const createWrapper = props => shallow(<LoginPage {...defaultProps} {...props}/>);

    it('snapshot testing', () => {
        const component = createWrapper();

        expect(component).toMatchSnapshot();
    });


});
