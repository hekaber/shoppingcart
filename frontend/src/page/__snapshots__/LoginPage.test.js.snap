// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Page snapshot testing 1`] = `
<div
  className="login-container"
>
  <Input
    name="username"
    placeholder="example@example.com hello    s"
    type="username"
  />
</div>
`;
